flight/PiOS/ChibiOS/os/hal/ports/STM32/STM32F4xx/stm32_registry.h

```diff
@@ -56,7 +56,7 @@
 
 #elif defined(STM32F405xx) || defined(STM32F415xx) ||                       \
       defined(STM32F407xx) || defined(STM32F417xx)
-#define STM32F40_41xxx
+// #define STM32F40_41xxx
```

Comment:
Defining STM32F405xx would be _strongly_ preferable to modifying Chibi sources.

Reply:
That one stems from STM32F40_41xxx being defined in the Revo makefile. Seppuku curiously doesn't. Should probably remove the one in the Makefile.

Reply:
Yes